rust/tvm/src/ir/attrs.rs

```diff
@@ -27,3 +27,19 @@ use tvm_macros::Object;
 pub struct BaseAttrsNode {
     pub base: Object,
 }
+
+impl BaseAttrsNode {
+    pub fn base<T: IsObject>() -> BaseAttrsNode {
+        BaseAttrsNode {
+            base: Object::base::<T>(),
+        }
+    }
+}
+
+impl Attrs {
```

Comment:
On the C++ side, most `Attrs` are constructed using their subclass, not as a `BaseAttrsNode`.  Is this a difference that we want to have between the C++ and Rust bindings?